src/Sets/Zonotope.jl

```diff
@@ -405,6 +404,11 @@ function split!(Z₁::Zonotope, Z₂::Zonotope, Z::Zonotope, j::Int)
     return Z₁, Z₂
 end
 
+function load_static_arrays()
+return quote
+
+using StaticArrays: SMatrix, SVector, MMatrix, MVector
```

Comment:
```suggestion
using .StaticArrays: SMatrix, SVector, MMatrix, MVector
```
I think you need to do this.